src/optimizely_helper.js

```diff
@@ -30,3 +30,11 @@ export function dispatchEvent({ url, params }) {
 
   return fetch(eventRequest);
 }
+
+/** 
+ * Generates a random 6 digit user id for demo purpose. For production uses, please use user Ids from your system
+ * or autogenerate a uuid.
```

Comment:
This addresses my point above to some degree, though I'm still wondering if we should try and use something better for default user ID generation. Testing the generation method below I see a decent number of collisions happening locally (~0.5-0.6% collision rate).